components/match2/commons/starcraft_starcraft2/match_group_input_starcraft.lua

```diff
@@ -469,7 +470,7 @@ function StarcraftMatchGroupInput._opponentInput(match)
 
 		--set initial opponent sumscore
 		opponent.sumscore =
-			tonumber(opponent.extradata.advantage or '') or ''
+			tonumber(opponent.extradata.advantage) or tonumber('-' .. (opponent.extradata.penalty or '')) or ''
```

Comment:
Mixing `number` with `string` fallback

Reply:
looking at the code we do not need the fallback at all